src/main/java/com/enderio/base/common/lang/EIOLang.java

```diff
@@ -163,6 +163,13 @@ private static Component enchantmentDescription(String enchantmentname, String d
     public static final MutableComponent SHOW_DETAIL_TOOLTIP = REGISTRATE.addLang("tooltip", EnderIO.loc("gui.show_advanced_tooltip"), "<Hold Shift>");
 
     // endregion
+    
+    // region Guidebook
+    
+    public static final Component GUIDEBOOK_TITLE = REGISTRATE.addLang("guidebook", EnderIO.loc("book_title"), "Book Title");
```

Comment:
Is all book content going to be done using language keys too? If so it might be worth adding a class dedicated to book lang

Reply:
From what I remeber, there are 2 options. Using language keys, or putting it in a different folder. But I'm not to sure on the best method. Botania does use the keys iirc.

Reply:
Alrighty, thats okay then, was just curious incase we ended up with a massive file, Thanks :)